source/extensions/filters/udp/udp_proxy/udp_proxy_filter.cc

```diff
@@ -276,7 +276,9 @@ UdpProxyFilter::ActiveSession::ActiveSession(ClusterInfo& cluster,
       idle_timer_(cluster.filter_.read_callbacks_->udpListener().dispatcher().createTimer(
           [this] { onIdleTimer(); })),
       udp_session_info_(
-          StreamInfo::StreamInfoImpl(cluster_.filter_.config_->timeSource(), nullptr)),
+          StreamInfo::StreamInfoImpl(cluster_.filter_.config_->timeSource(),
+                                     std::make_shared<Network::ConnectionInfoSetterImpl>(
+                                         addresses_.local_, addresses_.peer_))),
```

Comment:
Can you cover this change in a test somehow? Thanks.

/wait

Reply:
Added Unit Test